app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -106,18 +106,28 @@ fun HomeScreen(
         )
     }
 
-    FileRestrictionDialog(
-        featureFlagState = featureFlagNotificationViewModel.featureFlagState,
-        hideDialogStatus = featureFlagNotificationViewModel::dismissFileSharingDialog
-    )
-
     with(featureFlagNotificationViewModel.featureFlagState) {
+        if (showFileSharingDialog) {
+            FileRestrictionDialog(
+                isFileSharingEnabled = featureFlagNotificationViewModel.featureFlagState.showFileSharingDialog,
+                hideDialogStatus = featureFlagNotificationViewModel::dismissFileSharingDialog
+            )
+        }
+
         if (shouldShowGuestRoomLinkDialog) {
             GuestRoomLinkFeatureFlagDialog(
                 isGuestRoomLinkEnabled = isGuestRoomLinkEnabled,
                 onDismiss = featureFlagNotificationViewModel::dismissGuestRoomLinkDialog
             )
         }
+
+        if (shouldShowSelfDeletingMessagesDialog) {
+            SelfDeletingMessagesDialog(
+                isSelfDeletingMessagesEnabled = areSelfDeletedMessagesEnabled,
```

Comment:
change it to areSelfDeletedMessagesEnabled  or isSelfDeletingMessagesEnabled pls :"P